utils/connection/retry.go

```diff
@@ -130,7 +129,6 @@ func (p *RetryProfile) MakeGrpcRetryPolicyJSON() string {
 				"retryableStatusCodes": []string{
 					"UNAVAILABLE",
 					"DEADLINE_EXCEEDED",
-					"RESOURCE_EXHAUSTED",
```

Comment:
Can `RESOURCE_EXHAUSTED` be a temporary problem? If so, shouldn't we retry.
I understand that in the case this PR is solving, the error is permanent. But I'm not sure it is always the case for this error.

Reply:
I will revert this change for now. We can investigate more and make changes to this code later.